dotnet/Capstone/Controllers/BirdController.cs

```diff
@@ -71,35 +87,117 @@ public Bird getRandomBird()
             return randomBird;
         }
 
-        [HttpPost("/birds")]
-        public void createBird(Bird newBird)
+        [HttpPost("/lists/{listId}/addBird")]
+        public IActionResult createBird([FromBody] Bird newBird, int listId)
         {
-            //const string ErrorMessage = "An error occurred and user was not created.";
+            const string errorMessage = "An error occurred and a bird sighting was not created.";
+            
+            IActionResult result;
             try
             {
-                Bird bird = birdDao.createBird(newBird, newBird.name, newBird.description, newBird.imgUrl);
-                if (bird == null)
-                {
-                    Console.WriteLine(StatusCode(404));
-                }
-            }catch (DaoException )
+                Bird bird = birdDao.createBird(newBird, listId);
+
+                result = Created("", "");
+            }
+            catch (DaoException e)
+            {
+                result = BadRequest(new { message = e.Message });
+            }
+            catch (ArgumentException e)
+            {
+                result = BadRequest(new { message = e.Message });
+            }
+            catch
             {
-                Console.WriteLine(StatusCode(500));
+                result = BadRequest(new { message = errorMessage });
             }
 
+            return result;
+
         }
 
-        [HttpPut("/birds/{id}")]
-        public void editBird(Bird updatedBird, int id)
+        [HttpPut("/updateBird/{id}")]
```

Comment:
This id in this route needs removed. See format in [BirdService.js](https://github.com/ahdisease/Bird-Finder/blob/bird-sighting/vue/src/services/BirdService.js)

Reply:
Done